deployments/_bytecode_storage.json

```diff
@@ -39,7 +39,8 @@
b61283f88838901612708565b9550602087013591508082111561285557600080fd5b818701915087601f83011261286957600080fd5b81358181111561287857600080fd5b8860208260051b850101111561288d57600080fd5b6020830195508094505050506128a686604087016127ed565b905092959194509250565b60008060c083850312156128c457600080fd5b823567ffffffffffffffff8111156128db57600080fd5b6128e785828601612708565b92505061252a84602085016127ed565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052603260045260246000fd5b60006020828403121561293857600080fd5b61209b826124d7565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052601160045260246000fd5b60007fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff82036129a1576129a1612941565b5060010190565b6020808252818101839052600090604080840186845b87811015612a0a5773ffffffffffffffffffffffffffffffffffffffff806129e5846124d7565b168452806129f48785016124d7565b16848701525091830191908301906001016129be565b5090979650505050505050565b81810381811115610f2057610f20612941565b600082357fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff21833603018112612a5e57600080fd5b9190910192915050565b60005b83811015612a83578181015183820152602001612a6b565b50506000910152565b60008151808452612aa4816020860160208601612a68565b601f017fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe0169290920160200192915050565b602081528151602082015260006020830151610140806040850152612aff610160850183612a8c565b915060408501517fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe0858403016060860152612b3a8382612a8c565b9250506060850151612b64608086018273ffffffffffffffffffffffffffffffffffffffff169052565b50608085015173ffffffffffffffffffffffffffffffffffffffff811660a08601525060a085015173ffffffffffffffffffffffffffffffffffffffff811660c08601525060c085015160e085015260e0850151610100818187015280870151915050610120612bd78187018315159052565b90950151151593019290925250919050565b600060208284031215612bfb57600080fd5b5051919050565b600060208284031215612c1457600080fd5b813561209b816126ec565b60008083357fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe1843603018112612c5457600080fd5b83018035915067ffffffffffffffff821115612c6f57600080fd5b602001915036819003821315612c8457600080fd5b9250929050565b60008085851115612c9b57600080fd5b83861115612ca857600080fd5b5050820193919092039150565b7fffffffff000000000000000000000000000000000000000000000000000000008135818116916004851015612cf55780818660040360031b1b83161692505b505092915050565b8183823760009101908152919050565b60208152600061209b6020830184612a8c565b600060208284031215612d3257600080fd5b815167ffffffffffffffff811115612d4957600080fd5b8201601f81018413612d5a57600080fd5b8051612d686126b582612650565b818152856020838501011115612d7d57600080fd5b612d8e826020830160208601612a68565b95945050505050565b600060208284031215612da957600080fd5b815161209b816126ec565b80820180821115610f2057610f20612941565b60008251612a5e818460208701612a6856fea26469706673582212207ccb78ab9459cec277fd30a891a7ead0dc8de5c978c8cfe1d0b8835d98cc5ded64736f6c63430008110033"
```

Comment:
Why is the code of 1.0.0 replaced?
Maybe this is an error (= tried to deploy before the version was bumped)?

I would expect this PR to only add a new version but not change the code of an existing one

Reply:
The version did not get replaced. Just a comma was added at the end to be able to add the next version.